Cargo.toml

```diff
@@ -36,6 +36,11 @@ rand = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.94"
 sha2 = "0.10"
+ssz = { git = "https://github.com/alpenlabs/ssz-gen", tag = "v0.6.1" }
+ssz_derive = { git = "https://github.com/alpenlabs/ssz-gen", tag = "v0.6.1" }
+ssz_types = { git = "https://github.com/alpenlabs/ssz-gen", tag = "v0.6.1" }
+tree_hash = { git = "https://github.com/alpenlabs/ssz-gen", tag = "v0.6.1" }
+tree_hash_derive = { git = "https://github.com/alpenlabs/ssz-gen", tag = "v0.6.1" }
```

Comment:
These are "internal" deps (as in they are from `alpenlabs` github org). We should move them up with the `strata-*` deps.